packages/terra-clinical-application/README.md

```diff
@@ -0,0 +1,32 @@
+# Terra Clinical Application
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-clinical-application.svg)](https://www.npmjs.org/package/terra-clinical-application)
+[![Build Status](https://travis-ci.org/cerner/terra-ui.svg?branch=master)](https://travis-ci.org/cerner/terra-ui)
+
+The Application is a wrapper component around `terra-base` that provides clinical application-specific properties.
```

Comment:
It looks like children components need to add an `app` prop to render correctly. Discussed in person that you expected users to utilize specific components to handle this. Possibly add documentation regarding this?